nipy/modalities/fmri/experimental_paradigm.py

```diff
@@ -51,11 +52,16 @@ def __init__(self, con_id=None, onset=None, amplitude=None):
             try:
                 # this is only for backward compatibility:
                 #if con_id were integers, they become a string
+                warnings.warn(
+                    'The behavior will change,' +
+                    'prepending by a c will disappear in the future')
                 self.con_id = np.array(['c' + str(int(float(c)))
-                                        for c in con_id])
+                                         for c in con_id])
             except:
```

Comment:
What error are you trying to catch with the `except`?   Should try and avoid bare `except` if at all possible.